lib/practice_exercises.rb

```diff
@@ -1,13 +1,35 @@
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: o(n)
+# Space Complexity: o(1)
 def remove_duplicates(list)
-  raise NotImplementedError, "Not implemented yet"
+  #raise NotImplementedError, "Not implemented yet"
+  array = list.length
+  index_1 = 0
+  array.times do
+    if list[index_1] == list[index_1 + 1]
+      list.delete_at(index_1)
+    else 
+      index_1 +=1
+    end
+  end 
+  return list
 end
 
-# Time Complexity: ?
-# Space Complexity: ?
+# Time Complexity: o(n^2)
```

Comment:
This is actually O(mn) where m is the length of the strings and n is the number of strings.